### PR TITLE
fix - gui in buster

### PR DESCRIPTION
**Pull request by RobertLucian**
Attempt to fix the GUI problem in Buster. It looks like the `python-wxgtk2.8` package is no longer available in Buster and that causes all other packages to not get installed - specifically `python-wxgtk3.0` which is absolutely necessary for our GUI apps.

This is the error I was getting whenever a robot is being installed:
Screenshot 2019-06-26 14 41 02

With this fix, the `python-wxgtk3.0` and `wx3.0-i18n` packages get installed.

### CODE REVIEW

File: Troubleshooting_GUI/install.sh [hunk 1]
```diff
@@ -39,4 +39,4 @@ sudo chmod +x /home/pi/Desktop/Troubleshooting_Start.desktop
 
 delete_folder /home/pi/GoBox/Troubleshooting
 
-feedback "Done with TRoubleshooting"
\ No newline at end of file
+feedback "Done with TRoubleshooting"
```
Comment (CleoQc) [inline Troubleshooting_GUI/install.sh]: Since the file is already checked out, could you fix this typo here? (Capital R should be lowercase)

**Review by CleoQc — DISMISSED** [review]
Just a typo (that was already in there) to fix

File: Troubleshooting_GUI/install.sh (hunk 1 above)
Comment (RobertLucian) [reply Troubleshooting_GUI/install.sh]: Done.

**Review by CleoQc — APPROVED** [review]

**Comment by RobertLucian** [comment]
To be included in a future PR.

**Pull request closed without merging**